src/wallet/rpcwallet.cpp

```diff
@@ -2473,8 +2481,7 @@ Value zc_raw_receive(const json_spirit::Array& params, bool fHelp)
         throw runtime_error(
             "zcrawreceive zcsecretkey encryptednote\n"
             "\n"
-            "Scheduled for deprecation. This call will be removed in 1.0.\n"
-            "Decrypts encryptednote and checks if the coin commitments\n"
+            "DEPRECATED. Decrypts encryptednote and checks if the coin commitments\n"
```

Comment:
We may remove in beta 2 to get the task out of the way, so don't want to specify an exact release version.